pkg/products/monitoring/reconciler.go

```diff
@@ -273,7 +273,14 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 
 	phase, err = r.newAlertsReconciler().ReconcileAlerts(ctx, serverClient)
 	logrus.Infof("Phase: %s reconcilePrometheusRule", phase)
+	if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
+		events.HandleError(r.recorder, installation, phase, "Failed to reconcile alerts", err)
+		return phase, err
+	}
 
+	// creates an alert to check for the presents of sendgrid smtp secret
+	phase, err = resources.CreateSmtpSecretExists(ctx, serverClient, installation)
+	logrus.Infof("Phase: %s CreateSmtpSecretExistsRule", phase)
 	if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
 		events.HandleError(r.recorder, installation, phase, "Failed to reconcile alerts", err)
```

Comment:
Should we update the error message to specifically state it failed to create smtp  alert

Reply:
Sure why not